R/generate_simulated_data.R

```diff
@@ -341,6 +346,15 @@ generate_simulated_data <- function(r_in_weeks = # nolint
     )[(uot + 1):(uot + ot + ht)]
   }
 
+  # unit test to make sure these are equivalent
```

Comment:
I was thinking of this as a unit test in the actual testing suite, not a validation check in the data generation. But no reason not to keep this since it's fast.